src/docker-osx-dev

```diff
@@ -399,7 +399,7 @@ function configure_docker_machine {
   DOCKER_HOST_NAME="$DOCKER_MACHINE_NAME"
   DOCKER_HOST_USER=$(inspect_docker_machine "{{.Driver.SSHUser}}")
   DOCKER_HOST_IP=$(inspect_docker_machine "{{.Driver.IPAddress}}")
-  DOCKER_MACHINE_STORE_PATH=$(inspect_docker_machine "{{.StorePath}}")
+  DOCKER_MACHINE_STORE_PATH=$(inspect_docker_machine "{{.HostOptions.AuthOptions.StorePath}}")
```

Comment:
Hm, I just did a Google search for "docker machine StorePath" and many of the results, such as [this one](https://github.com/docker/machine/issues/843), had a `StorePath` at the top level, as well as one nested within `HostOptions.AuthOptions`. Why are there two? Which one is the right one to use?